[T3n] Need new Typical for Electric Heaters

YoannLe opened the pull request:
I'm working on design of boards for souliss based on ESP8266.
I've finish to test my version for electrics curtains controls and I will publish code and boards files soon.

Common goals for these designs:
- the boards must fit in standard electric wall boxes
- be supplied by 230V
- limit the number of components for DIY feasibility
- under 50€ by complete unit (logic and power board)
- common logic board and different power boards

I want to fork my power board to make a new board to control electric heater mode (eco, on, off...).

In France (and Europe) electrics heaters comes with 1 wire protocol called "Fil Pilote" and I need a new Typical to pilot this function. I'm not enough skilled to write this typical and I need help on this point.

Definition T33 Typical Electric Heater "Fil Pilote":

1 logical input with 6 states:
- Confort (ON)
- Confort -1°C (ON -1°)
- Confort -2°C (ON -1°)
- Eco (ON -3.5°)
- Hors-Gel (Frost Protect)
- Arrêt (OFF)

2 outputs:
Confort: O1=0 O2=0
Arrêt (OFF): O1=1 O2=0
Hors-Gel (Frost Protect): O1=0 O2=2
Eco: O1=1 O2=1
Confort -1°C (ON -1°): cyclic: 3 seconds O1=1 O2=1 / 297 seconds O1=0 O2=0 / 3 seconds O1=1 O2=1 / ....
Confort -2°C (ON -1°): cyclic: 7 seconds O1=1 O2=1 / 293 seconds O1=0 O2=0 / 3 seconds O1=1 O2=1 / ....

Commit:
Fix #195

Commit:
Added SHIFT

Commit:
Fix #194

Commit:
Fix #167

Commit:
Fix #188

Commit:
Minor improvements

Commit:
Minor patches

Commit:
Additional note to clarify use of WIFI parameters in ethUsrCfg.h

Commit:
Update T3n.cpp

Commit:
Update Typicals.h

Commit:
Merge branch 'stefanst-souliss' into patch7.1.1

plinioseniore commented:
You should open an issue to request a new feature and not a pull request
https://github.com/souliss/souliss/issues
https://github.com/souliss/souliss/wiki/Contribute

I've quickly read about the Fil Pilote, I guess that we first need to understand how to send command on that bus, or have you a library for?

YoannLe commented:
OK, thank you, I've corrected my mistake.That is not really a bus, just a wire that pilots heaters mode, I've added some schematics on my issue.
Regards
      De : Dario Di Maio notifications@github.com
 À : souliss/souliss souliss@noreply.github.com 
Cc : Yoann Lefebvre yoann.l@gmail.com 
 Envoyé le : Dimanche 1 novembre 2015 20h28
 Objet : Re: [souliss] [T3n] Need new Typical for Electric Heaters (#206)

You should open an issue to request a new feature and not a pull request
https://github.com/souliss/souliss/issues
https://github.com/souliss/souliss/wiki/ContributeI've quickly read about the Fil Pilote, I guess that we first need to understand how to send command on that bus, or have you a library for?—
Reply to this email directly or view it on GitHub.
